cobalt/black_box_tests/testdata/service_worker_persist_test_script.js

```diff
@@ -92,7 +92,7 @@ function test_successful_registration() {
 function test_persistent_registration() {
     console.log("test_persistent_registration()");
     navigator.serviceWorker.getRegistration(
-        '/bar/registration/scope')
+        '/bar/foo/nope')
```

Comment:
Perhaps try `baz` instead of `foo` for more cromulent code

Reply:
Ack